sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/implementation/directconnectivity/AddressResolverTest.java

```diff
@@ -211,7 +213,9 @@ private void TestCacheRefreshWhileRouteByPartitionKey(
 
         request.forceNameCacheRefresh = forceNameCacheRefresh;
         request.forcePartitionKeyRangeRefresh = forceRoutingMapRefresh;
-        request.getHeaders().put(HttpConstants.HttpHeaders.PARTITION_KEY, new PartitionKey("foo").toString());
+        PartitionKey pk = new PartitionKey("foot");
```

Comment:
What "foot" represents ?

Reply:
that's pk value name. changed back to foo.